### PR TITLE
@types/vuelidate: required, minLength - minor fixes.

### PR DESCRIPTION
**Pull request by janesser**
Please fill in this template.

- [x ] Use a meaningful title for the pull request. Include the name of the package modified.
- [x ] Test the change in your own code. (Compile and run.)
- [x ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x ] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:

- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [ ] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:

- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the `package.json`.

If removing a declaration:

- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Comment by janesser** [comment]
preps and tests happened earlier over here: https://github.com/janesser/vue-playground/commit/67f2f86de266f5cdfff8bc3f31190d5ec712b832

present test, checks this change on L195 - not exactly sure how it could pass earlier

File: types/vuelidate/tsconfig.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
             "DOM.Iterable",
             "ScriptHost"
         ],
-        "noImplicitAny": true,
+        "noImplicitAny": false,
```
Comment (RyanCavanaugh) [inline types/vuelidate/tsconfig.json:11]: No, we can't turn this off.

**Review by RyanCavanaugh — CHANGES_REQUESTED** [review]

**Comment by janesser** [comment]
In fact i have rushed over this errors

```
> npm test vuelidate

> definitely-typed@0.0.3 test
> node --enable-source-maps node_modules/@definitelytyped/dtslint/ types vuelidate

dtslint@0.2.25
Error: 
/home/jan/projs/DefinitelyTyped/types/vuelidate/vuelidate-tests.ts
   1:28  error  TypeScript@4.8, 4.9, 5.0, 5.1, 5.2, 5.3, 5.4, 5.5, 5.6, 5.7 compile error: 
Could not find a declaration file for module 'vuelidate'. '/home/jan/projs/DefinitelyTyped/node_modules/vuelidate/lib/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/vuelidate` if it exists or add a new declaration (.d.ts) file containing `declare module 'vuelidate';`                                           @definitelytyped/expect
  14:8   error  TypeScript@4.8, 4.9, 5.0, 5.1, 5.2, 5.3, 5.4, 5.5, 5.6, 5.7 compile error: 
Could not find a declaration file for module 'vuelidate/lib/validators'. '/home/jan/projs/DefinitelyTyped/node_modules/vuelidate/lib/validators/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/vuelidate` if it exists or add a new declaration (.d.ts) file containing `declare module 'vuelidate/lib/validators';`  @definitelytyped/expect
  63:37  error  TypeScript@4.8, 4.9, 5.0, 5.1, 5.2, 5.3, 5.4, 5.5, 5.6, 5.7 compile error: 
Parameter 'vm' implicitly has an 'any' type                                                                                                                                                                                                                                                                                                   @definitelytyped/expect

✖ 3 problems (3 errors, 0 warnings)

    at combineErrorsAndWarnings (/home/jan/projs/DefinitelyTyped/node_modules/@definitelytyped/dtslint/src/index.ts:259:26)
    at runTests (/home/jan/projs/DefinitelyTyped/node_modules/@definitelytyped/dtslint/src/index.ts:250:18)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at main (/home/jan/projs/DefinitelyTyped/node_modules/@definitelytyped/dtslint/src/index.ts:103:22)
```

need to figure where this has come from.

**Comment by DangerBotOSS** [comment]
## Formatting

The following files are not formatted:
1. types/vuelidate/vuelidate-tests.ts

Consider running `pnpm dprint fmt` on these files to make review easier.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 9848f82256501499ff099c846494a59e325d5f83
</p>

**Comment by janesser** [comment]
@RyanCavanaugh hope the merge-request is fine now - your feedback will be much appreciated.

**Review by RyanCavanaugh — APPROVED** [review]

**Comment by janesser** [comment]
Ready to merge

**Pull request merged**